go.mod

```diff
@@ -2,6 +2,8 @@ module sigs.k8s.io/cluster-api-operator
 
 go 1.17
 
+replace sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.1.1
```

Comment:
Why do we need this replace statement?

Reply:
This is because test framework imports CAPI like this https://github.com/kubernetes-sigs/cluster-api/blob/main/test/go.mod#L5, AWS has a similar replace https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/main/go.mod#L5